sbin/create-swap.sh

```diff
@@ -5,8 +5,23 @@ set -x
 
 # Calculate the ideal swap size.
 # There should be 32 GB of temporary file system size (RAM and swap combined).
-# First, we fnd the RAM size in GB by converting from KB.
+# First, we find the RAM size in GB by converting from KB.
 ram_gb=$(expr $(cat /proc/meminfo | grep MemTotal | awk {'print $2'}) / 1000000)
+root_size=$(df -h | grep -P '\/$' | awk '{print $4}')
+# If the root file system size is in gigabytes (not terabytes), we need to make sure there is enough space.
```

Comment:
"root file system size" is inaccurate. What we are looking at is "root file system available space"